src/Symfony/Component/Notifier/Message/SmsMessage.php

```diff
@@ -25,15 +25,17 @@ final class SmsMessage implements MessageInterface
     private $transport;
     private $subject;
     private $phone;
+    private $options;
 
-    public function __construct(string $phone, string $subject)
+    public function __construct(string $phone, string $subject, MessageOptionsInterface $options = null)
```

Comment:
This change should be removed and discussed in another PR.

Reply:
PR created: https://github.com/symfony/symfony/pull/39365